### DISCUSSION TITLE
video always autoplays and plays 1 video track, and 2 dup audio tracks at same time, 1 audio track  cannot be stopped

### DISCUSSION THREAD

**Discussion by AnnieSixgun** [mealie-recipes/mealie#2811] [Feature Request]
### First Check

- [X] This is not a feature request
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the Mealie documentation, with the integrated search.
- [X] I already read the docs and didn't find an answer.
- [X] This issue can be replicated on the demo site (https://demo.mealie.io/)

### What is the issue you are experiencing?

This is the code I used to add a video to a mealie recipe, which works on my linux mint system,   correctly. 
  
When I use this code, or any iteration of the code,   to add a video within a mealie recipe,   
mealie automatically plays the  video  track and  **two**  identical  audio  tracks....    as soon as the recipe is accessed.

Clicking pause on the video player  _stops the video track **and** the audio track synced to the video track_,    however,,    the second audio track continues to play to its end. 

IF...  I wait for the **non** synced audio track to finish playing,  the video track and its synced audio track , will start and stop correctly untill you re-access the recipe again,  and that starts the autoplay process again

The only difference in the code is the iframe source URL  ..

In Mealie I  upload the local video path to recipe ASSETS,,  and mealie creates a URL,  I manually trim the "image" data  and insert the URL as the iframe source,, or just enter a https web URL,   for an external path to the video..  
Both setups auto play the video as described above.

On my website the iframe source URL is the local path to the video,   or a https web URL  for an external path to a video.  

Example
    <iframe src="/api/media/recipes/0d76ea78-07bd-43ae-9170-d37fc07d14f2/assets/grits.mp4" ></iframe>
 
The mealie demo program has an example  at   
 `https://demo.mealie.io/recipe/how-to-make-the-best-grits-you-ve-ever-tasted`

### Steps to Reproduce

1. I upload the local video to Assets which gives me the location of the video with extra IMG info
    <img src="/api/media/recipes/0d76ea78-07bd-43ae-9170-d37fc07d14f2/assets/pot-roast.mp4" height="100%" width="100%"> </img>

2. then strip the image tag info from the data leaving
    "/api/media/recipes/0d76ea78-07bd-43ae-9170-d37fc07d14f2/assets/pot-roast.mp4"

3. then insert the location into the iframe code

    <iframe width="560" height="315"
    src="/api/media/recipes/0d76ea78-07bd-43ae-9170-d37fc07d14f2/assets/pot-roast.mp4">
     </iframe>

4. Now  copy and paste code into the recipe, and now I have the video in the recipe,,,

however.. 
each time I access the recipe, the video autoplays and plays  one video and two audio tracks at the same time. 

### Please provide relevent logs

Do not know how to do this

### Mealie Version

RC1 and Demo

### Deployment

Docker (Linux)

**Comment by AnnieSixgun** [comment]
Dec 7 2023 UPDATE ..........  
Added example of the problem _again_  to the mealie demo website. 
URL  https://demo.mealie.io/recipe/breakfast-grits

VIDEO _**auto plays**_ whenever you access the recipe,  and so far I can not keep it from auto starting ,,,  
HOWEVER, 
when the mp4 video and audio tracks  start playing on accessing  the recipe,
the tracks seem to play correctly , 

**EXCEPT**
 the program also ,, and almost concurrently,,  plays another copy of the  audio track on top of the exist playing audio track,   and one of the two playing audio tracks can not be paused ...

**IF,** you press pause, the _video_ and **1** of the 2 playing audio tracks,,  stops.
**IF,** you allow the still  playing audio track to finish,  and then press play , the mp4 now plays and pauses correctly ,,,
**UNTIL** you access the recipe again, and then the scenario repeats.

**Comment by boc-the-git** [comment]
Just to note, I opened your link to the demo site, on my mobile Chrome browser, and had no issues with auto playing or multiple audio tracks.

It played and paused on my command.

**Comment by AnnieSixgun** [comment]
Just tested chrome on my other machines and results;

2023-12-07
  TESTED - Video problem on the Mealie Demo system
            - Mealie Demo fails on 64bit systems listed below;

ASUS laptop, Intel I9
    OS - Linux Mint 21 Cinnamon, Tested Firefox, Chrome,                       -  Chrome works
    OS - Virtual Machine Win10,    Tested Firefox, Chrome, MS Edge     -  Chrome works

Custom Desktop – ASROCK – AMD Ryzen 9
    OS - Linux Mint 21 Cinnamon,  Tested Firefox, Chrome,                      -  Chrome works
    OS - Virtual Machine Win10,     Tested Firefox, Chrome, MS Edge    -  Chrome works

Toshiba Laptop – Intel I5
    OS - Windows 7 –  Tested Firefox, IE                                                           -  Chrome works

**Comment by michael-genson** [comment]
I'm going to convert this to a discussion as Mealie does not support video playback outside of standard HTML tags

**Reply by AnnieSixgun** [reply]
Michael, you state in your message   "outside of standard HTML tags" !  What's wrong the the code !!!
 <iframe src="/api/media/recipes/da61b560-39cd-49a3-8a32-723b4a417952/assets/grits.mp4" > </iframe>
